src/core/src/main/java/org/apache/jmeter/config/gui/ArgumentsPanel.java

```diff
@@ -441,10 +441,10 @@ private void moveDown() {
         // or the selected rows will be unselected
         int[] rowsSelected = table.getSelectedRows();
         GuiUtils.stopTableEditing(table);
-
-        if (rowsSelected.length > 0 && rowsSelected[rowsSelected.length - 1] < table.getRowCount() - 1) {
+        int selectedRowsCount = rowsSelected.length;
+        if (selectedRowsCount > 0 && rowsSelected[selectedRowsCount - 1] < table.getRowCount() - 1) {
             table.clearSelection();
-            for (int i = rowsSelected.length - 1; i >= 0; i--) {
+            for (int i = selectedRowsCount - 1; i >= 0; i--) {
```

Comment:
Frankly, I think the performance gains are negligible here, and the change makes the code slightly harder to follow